### PR TITLE
[MLU] add mlu kernel for fill_constant_batch_size_like

### PR DESCRIPTION
**Pull request by Ashelly1**
### PR types

New features

### PR changes

OPs

### Describe

add mlu kernel for fill_constant_batch_size_like

### CODE REVIEW

**Comment by Ashelly1** [comment]
fill_constant_batch_size_like op:
image

File: python/paddle/fluid/tests/unittests/mlu/test_fill_constant_batch_size_like_op_mlu.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                  data_type,
+                                  input_dim_idx=0,
+                                  output_dim_idx=0,
+                                  force_cpu=False):
+    return paddle.fluid.layers.fill_constant_batch_size_like(
+        input, shape, data_type, value, input_dim_idx, output_dim_idx,
+        force_cpu)
+
+
+class TestFillConstatnBatchSizeLike1(OpTest):
```
Comment (Ivorfeng) [inline python/paddle/fluid/tests/unittests/mlu/test_fill_constant_batch_size_like_op_mlu.py]: 参考npu 测例，多加几个测试情况

Comment (Ashelly1) [reply python/paddle/fluid/tests/unittests/mlu/test_fill_constant_batch_size_like_op_mlu.py]: 已参考npu测例，增加多个测试情况

**Review by ShawnNew — APPROVED** [review]
👍

**Review by qili93 — APPROVED** [review]
LGTM

**Pull request merged**